src/bin/ambit/cmd.rs

```diff
@@ -181,7 +203,41 @@ pub fn sync(dry_run: bool, quiet: bool, move_files: bool) -> AmbitResult<()> {
         total_syncs += 1;
         Ok(())
     };
-    let entries = get_config_entries()?;
+    let entries = if use_repo_config || !AMBIT_PATHS.config.exists() {
+        if !use_repo_config {
+            // Ask user if they want to search for repo config.
+            println!(
+                "No configuration file found in {}",
+                AMBIT_PATHS.config.path.display()
+            );
+            print!("Search for configuration in repository? [Y/n] ");
+            io::stdout().flush()?;
+            let mut answer = String::new();
+            io::stdin().read_line(&mut answer)?;
+            if let Some(answer) = answer.chars().next() {
+                if answer.to_lowercase().to_string() != "y" {
+                    println!("Cancelling sync...");
+                    return Ok(());
+                }
+            }
+        }
+        println!(
+            "Searching for {} in {}...",
+            CONFIG_NAME,
+            AMBIT_PATHS.repo.path.display()
+        );
+        let repo_config = match get_repo_config_path()? {
```

Comment:
Would this mean that it would be impossible for `ambit` to use a repo config in a case where there are multiple `config.ambit`? What should happen if `ambit` supplies a repo config that the user doesn't want? Asking the user if they want to use it or not won't fix this. Any ideas?

Reply:
I'm thinking something similar to
```
$ ambit sync
Recursively search repo for config? [Y/n] y
Config found: repo/ext/config.ambit. Use? [Y/n] n
Config found: repo/config.ambit. Use? [Y/n] y
```